src/NServiceBus.RabbitMQ/Sending/PoisonMessageForwarder.cs

```diff
@@ -1,37 +1,25 @@
 ﻿namespace NServiceBus.Transport.RabbitMQ
 {
-    using System;
     using System.Threading.Tasks;
     using global::RabbitMQ.Client.Events;
-    using Logging;
 
     class PoisonMessageForwarder
     {
-        static readonly ILog Logger = LogManager.GetLogger(typeof(PoisonMessageForwarder));
-
         readonly IChannelProvider channelProvider;
 
         public PoisonMessageForwarder(IChannelProvider channelProvider)
         {
             this.channelProvider = channelProvider;
         }
 
-        public Task ForwardPoisonMessageToErrorQueue(BasicDeliverEventArgs message, Exception ex, string errorQueue)
+        public Task ForwardPoisonMessageToErrorQueue(BasicDeliverEventArgs message, string errorQueue)
```

Comment:
On the other hand, I'd like to see if there is a way to move more of the poison message stuff back into this class instead of having it all in the message pump.

Once the message pump has found a poison message, I'd like to be able to just hand it off to something to take care of it.